app/src/main/java/com/gophillygo/app/HomeActivity.java

```diff
@@ -213,7 +212,15 @@ public void onPointerCaptureChanged(boolean hasCapture) {
     private void clickedGridItem(int position) {
         Log.d(LOG_LABEL, "clicked grid view item: " + position);
 
-        Intent intent = new Intent(this, PlacesListActivity.class);
-        startActivity(intent);
+        switch (position) {
+            case 0:
+                // go to events list
+                startActivity(new Intent(this, EventsListActivity.class));
+                break;
+            default:
+                // go to places list
+                // TODO: #18 filter list based on selected grid item
+                startActivity(new Intent(this, PlacesListActivity.class));
+        }
```

Comment:
Consider moving this helper function into `PlaceCategoryGridAdapter`

Reply:
If I were to move the click handler to the adapter, I would also create a callback on the activity here to call `startActivity`, so I'm not sure what that would gain.

Is the idea to improve the grid item mapping from position? When the remaining items are implemented with the filters, I expect this will get reworked.

Reply:
The idea was to keep the knowledge about what items in the grid go where located solely inside `PlaceCategoryGridAdapter`. You could keep the call to `startActivity` inside `HomeActivity` here and add a method to `PlaceCategoryGridAdapter` that maps position or view -> `Intent`?

Anyways, it was only a suggestion, it's probably fine as-is too.

Reply:
Sure, implementing `getItem` or `getItemId` on the adapter would be a good place to map out what to launch. I'll do that with the filter implementation for the other grid items.